.cbmc-batch/jobs/aws_array_list_clear/cbmc-batch.yaml

```diff
@@ -1,4 +1,4 @@
-jobos: ubuntu16
-cbmcflags: "--bounds-check;--pointer-check;--div-by-zero-check;--signed-overflow-check;--unsigned-overflow-check;--pointer-overflow-check;--undefined-shift-check;--float-overflow-check;--nan-check;--unwinding-assertions;--unwind;1"
-goto: aws_array_list_clear_harness.goto
+cbmcflags: "--bounds-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwind;1;--unwinding-assertions;--object-bits;8"
```

Comment:
The change to this file does not seem related to the PR. This change adds `--object-bits 8` to the command line flags and reorders the keys in this file, I would prefer to keep such changes separate (if they're necessary at all, i.e. the reordering)

Reply:
I'll remove this.  At some point it might be nice to go and remake all the `.yaml` files to use the latest template, so these changes don't keep slipping into PRs.

Reply:
Indeed, why do you actually commit the .yaml files? I think we stopped doing that in FreeRTOS. At least for us, those files get generated at CI time.

In fact, I remember that committing .yaml files caused a problem: somebody committed a .yaml file, then changed the Makefile that generated the yaml file. But because the .yaml file was already in the repo and therefore 'up-to-date', Make did not rebuild it under CI, so the CI used old flags.

I'd recommend removing all .yaml files from the repo if possible.